museum_of_dreams_project/settings.py

```diff
@@ -32,6 +38,7 @@
     "35.179.22.251",
     "museumofdreams.eu-west-2.elasticbeanstalk.com",
     "museumofdreamworlds.eu-west-2.elasticbeanstalk.com",
+    "localhost",
```

Comment:
I think it's best only to add localhost for local. Also DEBUG needs to be false in production